Random Notes by Martell

Not sure if I should #ifndef CYGWIN around the header,
or just move the android define up like currently

--- a/libbb/missing_syscalls.c.orig.c	2014-10-19 14:29:02.519274100 +0100
+++ a/libbb/missing_syscalls.c	2014-10-19 14:27:07.275058700 +0100
@@ -7,10 +7,11 @@
 //kbuild:lib-y += missing_syscalls.o
 
 /*#include <linux/timex.h> - for struct timex, but may collide with <time.h> */
+#if defined(ANDROID) || defined(__ANDROID__)
+
 #include <sys/syscall.h>
 #include "libbb.h"
 
-#if defined(ANDROID) || defined(__ANDROID__)
 pid_t getsid(pid_t pid)
 {
 	return syscall(__NR_getsid, pid);
